README.md

```diff
@@ -18,70 +18,11 @@
 
 `untool` is a JavaScript tool designed to streamline configuration and usage of other JavaScript tools. By default, it comes with a basic development and runtime environment for universal React applications. This environment is highly modular. Every one of its default modules is configurable and extensible - and entirely optional.
 
-`untool` is not really meant to be used on its own, but extended and built upon. To get everything you need to develop and deploy a production grade web application, please check out [`hops`](https://github.com/xing/hops).
+## Docs
 
-## Why?
+[Read the docs here](https://github.com/untool/untool/tree/master/DOCUMENTATION.md).
```

Comment:
Master is now the "cutting edge" documentation, so I think it should point to `v1.x` instead.
```suggestion
[Read the docs here](https://github.com/untool/untool/tree/v1.x/DOCUMENTATION.md).
```

Reply:
Okay, then I'll change that in the other READMEs, too.